src/legacy/core_plugins/kibana/public/dashboard/plugin.ts

```diff
@@ -38,8 +42,13 @@ import {
   HomePublicPluginSetup,
 } from '../../../../../plugins/home/public';
 import { SharePluginStart } from '../../../../../plugins/share/public';
-import { KibanaLegacySetup } from '../../../../../plugins/kibana_legacy/public';
+import {
+  AngularRenderedAppUpdater,
+  KibanaLegacySetup,
+} from '../../../../../plugins/kibana_legacy/public';
 import { createSavedDashboardLoader } from './saved_dashboard/saved_dashboards';
+import { createKbnUrlTracker } from '../../../../../plugins/kibana_utils/public/state_management/url/kbn_url_tracker';
+import { getQueryStateContainer } from '../../../../../plugins/data/public/query/state_sync/sync_query';
```

Comment:
this imports should be from the root of the plugins
`'../../../../../plugins/data/public`
Not sure why `eslint` rule didn't complain 🤔 ? kibana app overrides it?

Reply:
Ah good catch, I wasn't aware setting a new restricted paths rule on another level would override the additional ones. I made sure the top level rules are included (and also re-exported correctly).